inc/sitemaps/class-post-type-sitemap-provider.php

```diff
@@ -324,7 +320,11 @@ protected function get_excluded_posts() {
 		 *
 		 * @api array $posts_to_exclude The posts to exclude.
 		 */
-		$excluded_posts_ids = apply_filters( 'wpseo_exclude_from_sitemap_by_post_ids', array() );
+		$excluded_posts_ids = apply_filters(
+			'wpseo_exclude_from_sitemap_by_post_ids',
+			$this->get_page_for_posts_id() ? array( $this->get_page_for_posts_id() ) : array()
```

Comment:
This ternary is pretty hard to read. I would choose for declaring a variable `$post_to_exclude`

```php
$posts_to_exclude = array();
if ( $this->get_page_for_posts_id() ) {
    $posts_to_exclude = array( $this->get_page_for_posts_id() );
}
```